Examples/Tutorials/Tutorial-PcapFiles/main.cpp

```diff
@@ -88,6 +95,6 @@ int main(int argc, char* argv[])
 	pcapWriter.close();
 	pcapNgWriter.close();
 
-	// free reader memory because it was created by pcpp::IFileReaderDevice::getReader()
-	delete reader;
+	// No need to delete reader, unique_ptr will handle that
```

Comment:
no need this line

Reply:
Thank you @tigercosmos for your feedback. I appreciate your perspective and I'm glad we could reach a consensus. I've made the suggested changes and committed them. The updated code should now reflect the review issues we discussed. Please let me know if there's anything else that needs attention.